ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: openshift-e2e-test-qe
+  steps:
+  - ref: cucushift-pre
+  - ref: cucushift-e2e
```

Comment:
@liangxia could we put openshift-extended-test before cucushift-e2e because golang case will become more and ruby case become less in long term? 
cc @jhou1 @jianzhangbjz

Reply:
+1

Reply:
Sounds reasonable to me. Updated.